presto-kafka/src/main/java/io/prestosql/plugin/kafka/encoder/avro/AvroRowEncoder.java

```diff
@@ -73,58 +73,63 @@ private boolean isSupportedType(Type type)
         return type instanceof VarcharType || SUPPORTED_PRIMITIVE_TYPES.contains(type);
     }
 
+    private String currentColumnMapping()
```

Comment:
nit: this could have been separate commit

Reply:
FYI  i suggested to rename the commit instead